packages/lib/src/components/ErrorBoundary/utils.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  leaveBreadcrumb,
+  SentryBreadcrumbTypes,
+} from "../../utils/errorReporting";
+
+/**
+ * `refreshOnOldBundleError` is a utility function that refreshes the page if the error is due to an old bundle.
+ * This is useful when the user has an old version of the application and the bundle has changed.
+ * @param error - The error object
+ */
+const refreshOnOldBundleError = (error: Error) => {
+  if (error.message.includes("error loading dynamically imported module")) {
```

Comment:
Could we turn this string into a constant (and use the constant in the corresponding test too)?